Felienne/hedy
# [FIX] Makes font consistent between global editor and code snippets

**Description**
In this PR we fix that both the global editor and the code snippets share the same `fontFamily` to make sure special characters such as comma's are shown identical.

**Fixes**
This PR fixes #2622

**How to test**
Navigate to level 3, notice that the comma's in both the code snippets and the global editor are the same.

**Old view**
![Schermafbeelding 2022-04-28 om 14 51 37](https://user-images.githubusercontent.com/38155424/165756163-c408849a-9e01-45b3-b6fe-4afafe58c5d9.png)

**New view**
![Schermafbeelding 2022-04-28 om 14 51 22](https://user-images.githubusercontent.com/38155424/165756177-805eb658-75d4-4eb4-ae25-f4872a75850e.png)
